src/Symfony/Component/DependencyInjection/Compiler/RegisterServiceSubscribersPass.php

```diff
@@ -109,13 +118,13 @@ protected function processValue(mixed $value, bool $isRoot = false): mixed
                 $name = $this->container->has($type.' $'.$camelCaseName) ? $camelCaseName : $name;
             }
 
-            $subscriberMap[$key] = new TypedReference((string) $serviceMap[$key], $type, $optionalBehavior ?: ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE, $name);
+            $subscriberMap[$key] = new TypedReference((string) $serviceMap[$key], $type, $optionalBehavior ?: ContainerInterface::EXCEPTION_ON_INVALID_REFERENCE, $name, $attributes);
             unset($serviceMap[$key]);
         }
 
         if ($serviceMap = array_keys($serviceMap)) {
             $message = sprintf(1 < \count($serviceMap) ? 'keys "%s" do' : 'key "%s" does', str_replace('%', '%%', implode('", "', $serviceMap)));
-            throw new InvalidArgumentException(sprintf('Service %s not exist in the map returned by "%s::getSubscribedServices()" for service "%s".', $message, $class, $this->currentId));
+            throw new InvalidArgumentException(sprintf('Service "%s" not exist in the map returned by "%s::getSubscribedServices()" for service "%s".', $message, $class, $this->currentId));
```

Comment:
should be removed. That's a false positive of fabbot due to using a first sprintf above to handle a subpart of the message.